README.md

```diff
@@ -88,3 +88,19 @@ The patterns can be structured in roughly three different categories. Please cli
 * [EAV](More/EAV) [:notebook:](https://en.wikipedia.org/wiki/Entity%E2%80%93attribute%E2%80%93value_model)
 * [Repository](More/Repository)
 * [ServiceLocator](More/ServiceLocator) [:notebook:](http://en.wikipedia.org/wiki/Service_locator_pattern)
+
+## Localisation & Supported Languages
+
+| Code | Language      | Documentation |
+| ------------- |-------------| -----|
+| en | English | [Docs :notebook:](https://designpatternsphp.readthedocs.io/en/latest/README.html) |
+| ca | Catalan | [Docs :notebook:](https://designpatternsphp.readthedocs.io/en/latest/README.html) |
```

Comment:
That link is wrong: leads to 'en'

Reply:
You mean the link for Catalan docs? What is the slug for catalan?